watchfiles/main.py

```diff
@@ -91,6 +92,7 @@ def watch(
         force_polling: if `True`, always use polling instead of file system notifications, default is `None` where
             `force_polling` is set to `True` if the `WATCHFILES_FORCE_POLLING` environment variable exists.
         poll_delay_ms: delay between polling for changes, only used if `force_polling=True`.
+        recursive: if true, always look recursively inside folders, default is True.
```

Comment:
```suggestion
        recursive: if true, watch for changes in sub-directories recursively, otherwise watch only for changes in the top-level directory, default is `True`.
```
Is that correct?